import_and_save.py

```diff
@@ -0,0 +1,85 @@
+import click
+import csv
+from importlib import import_module
+
+def save_into_mysql(filename, username, password):
+    try:
+        module = import_module('MySQLdb')
+        db_name = 'cli_test'
+        mydb_conn = module.connect(user=username, passwd=password)
+        cur = mydb_conn.cursor()
+        cur.execute("""CREATE DATABASE IF NOT EXISTS """ + db_name)
+        mydb_conn = module.connect(user=username, passwd=password, db=db_name)
+        cur = mydb_conn.cursor()
+        csv_data = csv.reader(file(filename))
+        lst = ['Name', 'Age', 'Address']
+        table_name = "People"
+        createsqltable = """CREATE TABLE IF NOT EXISTS """ + table_name + " (" + " VARCHAR(250),".join(lst) + " VARCHAR(250))"
+        cur.execute(createsqltable)
+        for row in csv_data:
+            cur.execute('INSERT INTO People(Name,Age,Address)''VALUES(%s, %s, %s)',row)
+        mydb_conn.commit()
+        cur.close()
+    except:
```

Comment:
this is a bad practice, don't catch every exception here.. you should catch the exceptions you expect.

Reply:
you mean to say some specific exceptions?

Reply:
yes, you should catch only exceptions you expect. This makes harder to debug, you don't know if the issue is related to opening file/database operation.